src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -27,7 +27,10 @@
 public class Main extends GuiApplication {
 
   public static final String APP_NAME = "RCTab";
-  public static final String APP_VERSION = "1.4.0.alpha";
+
+  // TODO Sync version number with release.yml and build.gradle:
+  // github.com/BrightSpots/rcv/issues/662
+  public static final String APP_VERSION = "1.4.0-alpha";
```

Comment:
`jpackage` required in one of the OSs, I forget which, maybe all. It's standard semantic versioning to use a dash instead of a dot here: https://semver.org/

Reply:
Fair point. I think we're going to need to update `isVersionNewer` in this PR too though if you want to do this here now. Alternatively, you could leave it as-is and we can file another P1 issue to deal with this.

Reply:
https://github.com/BrightSpots/rcv/issues/700 i'll deal with it separately